xml/admin_rbd.xml

```diff
@@ -155,135 +155,180 @@
    After you create a &rbd;, you can use it like any other disk device: format
    it, mount it to be able to exchange files, and unmount it when done.
   </para>
-
-  <procedure>
-   <step>
-    <para>
-     Make sure your &ceph; cluster includes a pool with the disk image you want
-     to map. Assume the pool is called <literal>mypool</literal> and the image
-     is <literal>myimage</literal>.
-    </para>
-<screen>&prompt.cephuser;rbd list mypool</screen>
-   </step>
-   <step>
+  <para>
+   The <command>rbd</command> command defaults to accessing the cluster using
+   the &ceph; <literal>admin</literal> user account. This account has full
+   administrative access to the cluster. This runs the risk of accidentally
+   causing damage, similarly to logging into a Linux workstation as &rootuser;.
+   Thus, it is preferable to create user accounts with fewer privileges and use
+   these accounts for normal read/write &rbd; access.
+  </para>
+   
+  <sect2 xml:id="ceph-rbd-creatuser">
+   <title>Creating a &ceph; User Account</title>
+   <para>
+    To create a new user account, use the <command>ceph</command> command with the
+    <command>auth get-or-create</command> subcommand and Monitor and &osd;
+    capabilities:
+   </para>
+<screen>&prompt.cephuser;ceph auth get-or-create client.<replaceable>ID</replaceable> mon 'profile rbd' osd 'profile <replaceable>profile name</replaceable> [pool=<replaceable>pool-name</replaceable>][, profile ...]' mgr 'profile rbd [pool=<replaceable>pool-name</replaceable>]'</screen>
+   <para>
+    For example, to create a user called <replaceable>qemu</replaceable> with read-write
+    access to the pool <replaceable>vms</replaceable> and read-only access to the pool
+    <replaceable>images</replaceable>, execute the following:
+   </para>
+<screen>ceph auth get-or-create client.<replaceable>qemu</replaceable> mon 'profile rbd' osd 'profile rbd pool=<replaceable>vms</replaceable>, profile rbd-read-only pool=<replaceable>images</replaceable>' mgr 'profile rbd pool=<replaceable>images</replaceable>'</screen>
+   <para>
+    The output from the <command>ceph auth get-or-create</command> command will
+    be the keyring for the specified user, which can be written to
+    <filename>/etc/ceph/ceph.client.<replaceable>ID</replaceable>.keyring.</filename>
+   </para>
+   <note>
     <para>
-     Map the image to a new block device.
+     The user ID can be specified when using the <command>rbd</command> command by providing the
+     optional <command>--id</command> <replaceable>ID</replaceable> argument. 
     </para>
-<screen>&prompt.cephuser;rbd map --pool mypool myimage</screen>
-    <tip>
-     <title>User Name and Authentication</title>
+   </note>
+   <para>
+    For more details on managing &ceph; user accounts, refer to the
+    <link
+     xlink:href="https://docs.ceph.com/docs/nautilus/rados/operations/user-management/">
+     official &ceph; documentation</link>.
```

Comment:
```suggestion
    For more details on managing &ceph; user accounts, refer to <xref linkend="cha-storage-cephx"/>.
```

Reply:
OK, this is my fault. I have read all the SES docs from beginning to end and should have remembered there was a section on this already. 

In my defence, I merely added the link that was in the bug. :-(